automl/automl-script/script.whizzml

```diff
@@ -94,7 +111,10 @@
         exc-models (config "excluded-models"))
     (if reuse-resources
       (get-unsupervised-models automl-execution exc-models)
-      (create-unsupervised-models train-split-dataset exc-fields exc-models))))
+      (create-unsupervised-models train-split-dataset
+                                  exc-fields
+                                  exc-models
+                                  shallow-search))))
```

Comment:
Is this defined somewhere else? or is it a typo?

Reply:
shallow-search I mean

Reply:
It's the name of the input parameter